test/smoke-fort/flang-445882/doit.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+save_status() { tval=$? && (($tval!=0)) && rval=$tval; }; rval=0
+echo "ZZZZZZZZZZ Derived Type Pointer"   # marker for extract-tests
```

Comment:
Nit : Is this ZZZZZZZZ intentional?

Reply:
Yes.  It is the separator for extract-tests if there are multiple tests combined in a single log file.